README.md

```diff
@@ -136,4 +136,69 @@ export default {
 		})
 	}
 };
-```
\ No newline at end of file
+```
+
+### allowedRoles
+
+An array of strings containing the roles allowed to access the app. For e.g. if the app needs to be restrictied it's possible to use a [custom role](https://docs.microsoft.com/en-us/azure/static-web-apps/authentication-authorization). *Notice* this only supports securing the complete app. If specific routes should be secured it's best to implement custom authentication within the app itself.
```

Comment:
I wonder if it would be better to link docs on role configuration instead of the general auth docs:

```suggestion
An array of strings containing the roles allowed to access the app. For example, if the app needs to be restricted, it's possible to use a [custom role](https://docs.microsoft.com/en-us/azure/static-web-apps/configuration#securing-routes-with-roles). **Note:** this only supports securing the complete app. If specific routes should be secured, you should implement custom authentication within the app itself.
```